doc/README.md

```diff
@@ -1,13 +1,15 @@
-# Contributing to the Nixpkgs manual
+# Contributing to the Nixpkgs reference manual
 
-This directory houses the sources files for the Nixpkgs manual.
+This directory houses the sources files for the Nixpkgs reference manual.
 
-You can find the [rendered documentation for Nixpkgs `unstable` on nixos.org](https://nixos.org/manual/nixpkgs/unstable/).
-The rendering tool is [nixos-render-docs](../pkgs/tools/nix/nixos-render-docs/src/nixos_render_docs), sometimes abbreviated `nrd`.
+Going forward, it should only contain [reference](https://nix.dev/contributing/documentation/diataxis#reference) documentation.
+For tutorials and guides, contribute to <https://nix.dev/> instead.
```

Comment:
Where do the _explanations_ go?

Reply:
I've seen at least two kinds of explanations:
- Why is something the way it is: contributor and in-code documentation
- How does something work / why is it meaningful or preferrable: Concepts (nix.dev)

Reply:
Added a mention that explanations should go to nix.dev: 83423d2589901dd4603fcf3af2573f92b8cbe7a4